File: 백준/Bronze/1546. 평균/평균.java
import java.io.*;
import java.util.StringTokenizer;
class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        double max = 0;
        double total = 0;
        int N = Integer.parseInt(br.readLine());

        double[] arr = new double[N];
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < N; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
            if (arr[i] > max) max = arr[i];
        }

        for (int i = 0; i < N; i++) {
            arr[i] = arr[i] / max * 100;
            total += arr[i];
        }
        System.out.print(total / N);
    }
}

File: 백준/Bronze/1546. 평균/평균.py
# M: 본인 점수 중 최댓값
# 모든 점수를 점수/M*100으로 고침
# 새로운 평균 구하기

# N 입력받기(시험 본 과목의 개수)
# 현재 성적 N개 입력받아 배열에 저장
n = int(input())
arr = list(map(int, input().split()))
m = max(arr)
for i in range(n):
    arr[i] = arr[i] / m * 100
print(sum(arr) / len(arr))
    